packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -3480,6 +3484,14 @@ class InputDecoration {
   /// a default height based on text size.
   final BoxConstraints? constraints;
 
+  /// Optional widget that appears below the [InputDecorator.child] and the border.
+  /// If non-null, the border's color animates to red and the [helperText] is
+  /// not shown.
+  ///
+  /// If both [errorText] and [error] are specified, then [error]
```

Comment:
The `errorText` doc should get a mention of this field too.

For a good example to follow, you can borrow the wording from `labelText`:
> If a more elaborate label is required, consider using [label] instead. Only one of [label] and [labelText] can be specified.